### PR TITLE
Added seqkit/fq2fa module

### PR DESCRIPTION
**Pull request by d-jch**
## PR checklist

Closes #XXX 

- [X] This comment contains a description of changes (with reason).
- [X] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [X] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [X] Add a resource `label`
- [X] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [X] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/seqkit/fq2fa/main.nf [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    conda "bioconda::seqkit=2.5.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/seqkit:2.5.0--h9ee0642_0':
+        'biocontainers/seqkit:2.5.0--h9ee0642_0' }"
+
+    input:
+    tuple val(meta), path(fastq)
+
+    output:
+    tuple val(meta), path("*.fa.gz"), emit: fasta
```
Comment (heuermh) [inline modules/nf-core/seqkit/fq2fa/main.nf:14]: What do you think about adding support for zstandard compression (`.fa.zst`) here?

**Review by nvnieuwk — APPROVED** [review]
LGTM!!

File: modules/nf-core/seqkit/fq2fa/main.nf (hunk 1 above)
Comment (d-jch) [reply modules/nf-core/seqkit/fq2fa/main.nf:14]: looks good

**Review by SPPearce — CHANGES_REQUESTED** [review]
Why are the tests failing?

**Review by SPPearce — APPROVED** [review]

**Pull request merged**